### PR TITLE
Add support PostCSS extensions (.sss, .pcss, .postcss)

### PR DESCRIPTION
**Pull request by shashkovdanil**

### CODE REVIEW

**Comment by shashkovdanil** [comment]
cc @timneutkens

**Comment by shashkovdanil** [comment]
cc @timneutkens

**Comment by timneutkens** [comment]
I don't think we should add this.

**Comment by shashkovdanil** [comment]
@timneutkens, why? It does not break anything, it just gives you the opportunity to use SugarSS in the project. Why then add postcss support to this plugin?

**Comment by ai** [comment]
@timneutkens `.pcss` is official extension PostCSS (when you use plugins with custom syntax). GitHub and many IDEs (or text editor plugins) support it.

File: packages/next-css/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "repository": "zeit/next-plugins",
   "dependencies": {
     "css-loader": "^2.1.0",
-    "extract-css-chunks-webpack-plugin": "^3.3.2",
+    "extract-css-chunks-webpack-plugin": "3.3.2",
```
Comment (iamstarkov) [inline packages/next-css/package.json]: why did you lock the version?

Comment (shashkovdanil) [reply packages/next-css/package.json]: My mistakes, sry, fixed

**Comment by shashkovdanil** [comment]
cc @timneutkens, @iamstarkov

**Comment by iamstarkov** [comment]
I can see a benefit in adding support for postcss, either it is done by adding support for it in next-css or as an extra plugin

**Comment by pajasevi** [comment]
+1 for this PR, it would really help with syntax highlighting and it does not break anything.

**Comment by timneutkens** [comment]
I'm going to close this in favor of https://github.com/zeit/next.js/issues/8626

Thanks for your contribution 🙏

**Pull request closed without merging**